src/chart/components/y_axis/price_labels/y-axis-labels.model.ts

```diff
@@ -61,7 +61,7 @@ export const LabelsGroups = {
  * - ...
  * Anything but the base labels which are generated in other component {@link YAxisBaseLabelsModel}
  */
-export class YAxisLabelsModel extends ChartBaseElement {
+export class FancyYAxisLabelsModel extends ChartBaseElement {
```

Comment:
Fancy? Not sure about name

Reply:
It's a way better than before anyway

Reply:
Every time I came back to this code, It's always hard to me to distinguish regular y-axis labels and this `badge` labels. Fancy indicates `badge`, `plain` and `rectangle labels`